src/reducers/requests/posts.js

```diff
@@ -46,6 +46,17 @@ function getPostThread(state = initialRequestState(), action) {
     );
 }
 
+function getPostThreadWithRetryAttempt(state = 0, action) {
+    switch (action.type) {
+    case PostTypes.GET_POST_THREAD_WITH_RETRY_ATTEMPT:
+        return state + 1;
```

Comment:
I just don't see it being used anywhere (not sure if it's going to be used in RN) and it's going to be very racy if you're opening and closing a bunch of different threads rapidly